tests/attr/layer/test_grad_cam.py

```diff
@@ -22,7 +22,7 @@ def test_simple_input_non_conv(self) -> None:
     def test_simple_multi_input_non_conv(self) -> None:
         net = BasicModel_MultiLayer(multi_input_module=True)
         inp = torch.tensor([[0.0, 6.0, 0.0]], requires_grad=True)
-        self._grad_cam_test_assert(net, net.relu, inp, ([21.0], [21.0]))
+        self._grad_cam_test_assert(net, net.multi_relu, inp, ([21.0], [21.0]))
```

Comment:
Initially, the relu attribute was either single input or multi-input module based on a flag. This didn't work well with JIT scripting, since it seems to expect a module attribute to have a consistent signature in all paths. I changed it to name the multi-input ReLU module differently which resolved the scripting issue.